src/style.css

```diff
@@ -1,3 +1,73 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  height: 60px;
+  display: flex;
+  background-color: #ffff;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.logo_link {
+  height: 40px;
+  width: 40px;
+  display: block;
+  margin-left: 25px;
+}
+
+.nav_link {
+  height: 15px;
+  color: #000;
+  position: relative;
+  font-weight: bold;
+  font-family: Roboto, sans-serif;
```

Comment:
It's a good practice to include fallback fonts in case the primary font is not available. For example, 'font-family: Roboto, Arial, sans-serif;' ensures that if 'Roboto' cannot be loaded, the browser will try 'Arial' next, and if that fails, it will fall back to the default sans-serif font of the user's system.

Reply:
👍